lib/rspec/matchers/power_assert_matchers.rb

```diff
@@ -54,11 +54,5 @@ module PowerAssertMatchers
 require_relative 'power_assert_matchers/version'
 
 module PowerAssert
-  INTERNAL_LIB_DIRS[RSpec::Matchers::PowerAssertMatchers] = File.dirname(
-    File.dirname(
-      File.dirname(
-        caller_locations(1, 1).first.path
-      )
-    )
-  )
+  INTERNAL_LIB_DIRS[RSpec::Matchers::PowerAssertMatchers] = File.dirname(caller_locations(1, 1).first.path, 3)
```

Comment:
https://docs.rubocop.org/rubocop/cops_style.html#stylenestedfiledirname
https://bugs.ruby-lang.org/issues/12194

I overlooked this NEWS 😮